website/docs/docs/collaborate/govern/model-versions.md

```diff
@@ -9,32 +9,76 @@ description: "Version models to help with lifecycle management"
 This functionality is new in v1.5.
 :::
 
-API versioning is a _complex_ problem in software engineering. It's also essential. Our goal is to _overcome obstacles to transform a complex problem into a reality_.
+Versioning APIs is a challenging problem in software engineering. The goal of model versions is not to make the problem go away, or pretend it's easier than it is. Rather, we want dbt to provide tools that make it possible to tackle this problem, thoughtfully and head-on, and to develop standard patterns for solving it.
 
 ## Related documentation
 - [`versions`](resource-properties/versions)
 - [`latest_version`](resource-properties/latest-version)
 - [`include` & `exclude`](resource-properties/include-exclude)
+- [`ref` with `version` argument](ref#versioned-ref)
 
 ## Why version a model?
 
-If a model defines a ["contract"](model-contracts) (a set of guarantees for its structure), it's also possible to change that model's contract in a way that "breaks" the previous set of parameters.
+If a model defines a ["contract"](model-contracts) (a set of guarantees for its structure), it's also possible to change that model's structure in a way that "breaks" the previous set of guarantees.
 
-One approach is to force every model consumer to immediately handle the breaking change when it's deployed to production. While this may work at smaller organizations or while iterating on an immature set of data models, it doesn’t scale well beyond that.
+One approach is to force every model consumer to immediately handle the breaking change as soon as it's deployed to production. This is actually the appropriate answer at many smaller organizations, or while rapidly iterating on a not-yet-mature set of data models. But it doesn’t scale well beyond that.
 
-Instead, the model owner can create a **new version**, during which consumers can migrate from the old version to the new.
+Instead, for mature models at larger organizations, the model owner can create a **new version**, during which consumers can migrate from the old version to the new.
 
-In the meantime, anywhere that model is used downstream, it can be referenced at a specific version.
+In the meantime, anywhere that model is used downstream, it can continue to be referenced at a specific version.
+
+In the future, we intend to also add support for **deprecating models**. Taken together, model versions and deprecation offer a pathway for _sunsetting_ and _migrating_. In the short term, avoid breaking everyone's queries. Over the longer term, older & unmaintained versions go away—they do **not** stick around forever.
+
+## When should you version a model?
+
+Many changes to a model are not breaking, and do not require a new version! Examples include adding a new column, or fixing a bug in modeling logic.
+
+By enforcing a model's contract, dbt can help you catch unintended changes to column names and data types that could cause a big headache for downstream queriers.
```

Comment:
True, but nothing to do with versioning on its own. Is the implicit recommendation here "when you break your contract, you should bump versions"?